lib/Target/OpenFhePke/OpenFhePkeTemplates.h

```diff
@@ -28,6 +28,42 @@ using PublicKeyT = PublicKey<DCRTPoly>;
 )cpp";
 // clang-format on
 
+// clang-format off
+constexpr std::string_view kWeightsPreludeTemplate = R"cpp(
+#include <fstream>
+#include <map>
+#include <string>
+#include <vector>
+
+#include "include/cereal/archives/portable_binary.hpp" // from @cereal
+#include "include/cereal/cereal.hpp" // from @cereal
```

Comment:
This implies that the frontend, if it would like to use this flag for an openfhe backend, requires an include and shared lib for cereal when JIT-compiling, yes? Maybe file an issue for that, though it seems low priority for the moment.

Reply:
https://github.com/google/heir/issues/1528